frontend/src/routes/Search/Details/DetailsPage.tsx

```diff
@@ -304,6 +347,7 @@ export default function DetailsPage() {
         />
       )}
       <Outlet context={searchDetailsContext} />
+      {pluginModal}
```

Comment:
nits: modal ideally should be top most component in the DOM.

Reply:
makesense